### ISSUE TITLE
is viewport move or zoom causing all children updateTransform?

### ISSUE THREAD

**Issue by lancety** [davidfig/pixi-viewport#388]
I was looking for perfromance improvement, because my pixi project updateTransform takes long time, then I saw https://codepen.io/sukantpal/pen/rNxYXaX?editors=0010   which is using "renderTexture bind" to 'map' a part of pixi stage display objects to the canvas, which can move the view without actually moving stage, which avoid unnecessary updateTransform. 

But from what I found when debugging https://davidfig.github.io/pixi-viewport/,  all the rects in the viewport have triggered updateTransform, also cannot find in this repo mentioned "renderTexture" anywhere, so does it means this lib is not using the renderTexture approach from  the first link? @ShukantPal @davidfig 

I initially came from this post https://javascript.plainenglish.io/inside-pixijs-projection-system-897872a3dc17

Thanks for view and reply.

**Comment by ShukantPal** [comment]
@lancety Yes, this repo is not using that technique at the moment. I might come back in the future and do the optimization for this package but wouldn't hold breathe for that to happen soon :)

**Comment by lancety** [comment]
it is really smart idea to render things that way.  half of my frame time is spending on updateTransform. Looking forward for this improvement. 
BTW, is the main issue we cannt do it because the filter?

**Comment by ShukantPal** [comment]
The main blocker is interactions become broken after doing that so you have patch the interaction system in PixiJS or use a custom EventBoundary using the new events system in PixiJS. I believe issues with filtering have been fixed in PixiJS core now.

**Comment by davidfig** [comment]
@ShukantPal To get this working we'd need to move to pixi's new interaction system? This seems a really good optimization.

**Comment by ShukantPal** [comment]
@davidfig Yes I designed the events system such that here we could create a custom event bridge that translates coordinates into world space.

**Comment by ShukantPal** [comment]
@davidfig Yes I designed the events system such that here we could create a custom event bridge that translates coordinates into world space.

**Comment by ShukantPal** [comment]
@davidfig Yes I designed the events system such that here we could create a custom event bridge that translates coordinates into world space.

**Comment by lancety** [comment]
hi @ShukantPal, in my project it doesnt need require processing events such as clicking sprites. 
I used some filters and pixi-spine. 
Wondering if there are any other known issues you found when using the projection matrix approach at here https://codepen.io/sukantpal/pen/ZEQvKBB?editors=0010
thanks for suggestions :)


my pixi packages detail
    "@pixi/filter-outline": "^4.1.3",
    "@pixi/particle-emitter": "^5.0.2",
    "@pixi/picture": "^3.0.5",
    "pixi-projection": "^0.4.4",
    "pixi-spine": "^3.0.4",
    "pixi.js": "^6.1.2",
